examples/gltf/app.js

```diff
@@ -101,6 +61,19 @@ function loadGLTF(urlOrPromise, gl, options = DEFAULT_OPTIONS) {
   });
 }
 
+function loadModelIndex(modelSelector) {
```

Comment:
Just nits: 
* This mixes logic and UI... Can we split into two functions?
* `Index` is often used for small integers, it threw me off a little, maybe `list` is better?

```
const modelList = loadModelList(); // Does not know about DOM...
addModelsToDropdown(modelList); // Does not know how to load the list...
```